yoyodyne/models/__init__.py

```diff
@@ -78,3 +79,16 @@ def add_argparse_args(parser: argparse.ArgumentParser) -> None:
         default="attentive_lstm",
         help="Model architecture. Default: %(default)s.",
     )
+    parser.add_argument(
+        "--tie_embeddings",
+        action="store_true",
+        default=defaults.TIE_EMBEDDINGS,
+        help="Shares embeddings for the source and target vocabularies. "
+        "This should always be set to true for pointer generator models.",
+    )
+    parser.add_argument(
+        "--no-tie_embeddings",
+        action="store_false",
+        dest="tie_embeddings",
+        help="Separates embeddings for the source and target vocabularies.",
```

Comment:
To fit style this should be: `no_tie_embeddings`. And I take the stance that you don't need to put a docstring for the negative ones. argparse is smart and groups the positive and negative pairs with the same destination together, so there's no ambiguity there.